src/test/send_images.cpp

```diff
@@ -66,14 +67,16 @@ int main( int argc, char **argv )
     msg.step = width * 3;
 
     int count = 0;
+    std::default_random_engine seed;
```

Comment:
The engine isn't a seed. It's the random number generator.
```suggestion
    std::default_random_engine random_generator;
```